autotest/ogr/ogr_fgdb.py

```diff
@@ -2129,3 +2129,27 @@ def test_ogr_fgdb_utc_datetime():
     f = lyr.GetNextFeature()
     # Check that the timezone +00 is present
     assert f.GetFieldAsString('EditDate') == '2020/06/22 07:49:36+00'
+
+
+def test_ogr_fgdb_alias():
```

Comment:
this test should be moved to ogr_openfilegdb.py.   ogr_fgdb.py is only for the SDK based driver. And this will save you the funny play with driver deregisteration :-)